common/perception_utils/test/src/test_object_classification.cpp

```diff
@@ -61,3 +61,114 @@ TEST(object_classification, test_getHighestProbLabel)
     EXPECT_EQ(label, ObjectClassification::CAR);
```

Comment:
![image](https://user-images.githubusercontent.com/20228327/196141866-84f653c3-5478-4e8b-9f87-65c70a34cbca.png)

pre-commit failed. Don't you know how to fix it on a github browser or locally with a command?

Reply:
It seems to be due to unnecessary line break in the code.
I fixed it in local and push later.

Reply:
I meant there is a auto-fix command in both github browser and local command.

Reply:
Sorry, I forgot `pre-commit run -a` command before git push.
(But I still had to fix some error manually)

Reply:
Got it.

`pre-commit install` which enables running pre-commit when committing every time is also useful.